modules/integration_aws-efs/conf/06-burst-cedit-balance.yaml

```diff
@@ -0,0 +1,14 @@
+module: "AWS EFS"
+name: "Burst Credit Balance"
+filtering: "filter('namespace', 'AWS/EFS')"
+value_unit: "credits"
+transformation: ".mean(over='5m')"
+tip: "See https://docs.aws.amazon.com/efs/latest/ug/performance.html#bursting"
+signals:
+  signal:
+    metric: "BurstCreditBalance"
+    filter: "filter('stat', 'min')"
```

Comment:
Doesn't seems to have min values.
Do you mean : filter: "filter('stat', 'mean')" ?

Reply:
replace by `lower`